### PR TITLE
Added libInformation and readerInformation to the python bindings

### PR DESCRIPTION
**Pull request by Nokse22**

### CODE REVIEW

**Comment by mwestphal** [comment]
That looks nice, could you add tests for it ?

Please review @snoyer @Meakk

**Comment by Nokse22** [comment]
> That looks nice, could you add tests for it ?

Yes, adding them.

File: python/F3DPythonBindings.cxx [hunk 1]
```diff
@@ -334,7 +334,33 @@ PYBIND11_MODULE(pyf3d, module)
...
     .def_static(
       "autoload_plugins", &f3d::engine::autoloadPlugins, "Automatically load internal plugins")
-    .def_static("get_plugins_list", &f3d::engine::getPluginsList);
+    .def_static("get_plugins_list", &f3d::engine::getPluginsList)
+    .def_static("get_lib_info", &f3d::engine::getLibInfo, py::return_value_policy::reference)
+    .def_static("get_readers_info", &f3d::engine::getReadersInfo);
+
+  // libInformation
+  py::class_<f3d::engine::libInformation>(module, "LibInformation")
+    .def(py::init<>())
```
Comment (snoyer) [inline python/F3DPythonBindings.cxx:343]:
```suggestion
```
Should not be needed (same for `ReaderInformation`) unless we need to create this instances from python code for some reason?

Comment (Nokse22) [reply python/F3DPythonBindings.cxx]: Ok

**Comment by Nokse22** [comment]
I just check the type in the tests, let me know if I need to check the actual content.

**Comment by snoyer** [comment]
> I just check the type in the tests, let me know if I need to check the actual content.

I'd say it's not necessary to check for exact content but I'd check for non-emptiness in addition to type where applicable (`assert isinstance(xs, list) and xs` instead of just `assert isinstance(xs, list)` for lists and same for `dict` and `str` values)

Also you could rename `test_plugins.py` to `test_engine.py` (should have been done way back when we started using `pytest`, better late than never) and add your tests in there to limit the number of files.

**Review by mwestphal — APPROVED** [review]

**Comment by mwestphal** [comment]
@snoyer please review

**Comment by snoyer** [comment]
looks good

**Pull request merged**